src/containers/PaymentMethodsPage/PaymentMethodsPage.js

```diff
@@ -68,18 +154,20 @@ export const PaymentMethodsPageComponent = props => {
       }`
     : null;
 
-  const initalValuesForStripePayment = { name: userName };
+  const initalValuesForStripePayment = { name: userName, country: 'FI' };
```

Comment:
Hard-coded 'FI' sounds a bit bad for the template app - unless it's in the config.js.